auth_mellon_config.c

```diff
@@ -1273,6 +1279,37 @@ static const char *am_set_invalidate_session_slots(cmd_parms *cmd,
     return NULL;
 }
 
+/* This function handles the MellonDisabledCurlExpectLogic configuration directive.
+ * This directive can be set to "on" or "off" (default).
+ *
+ * Parameters:
+ *  cmd_parms *cmd       The command structure for this configuration
+ *                       directive.
+ *  void *struct_ptr     Pointer to the current directory configuration.
+ *  const char *arg      The string argument following this configuration
+ *                       directive in the configuraion file.
+ *
+ * Returns:
+ *  NULL on success or an error string if the argument is wrong.
+ */
+static const char *am_set_curl_expect_logic_slots(cmd_parms *cmd,
```

Comment:
This function would need to be changed to am_set_send_expect_slots() if my other comment is accpeted